test/hotspot/jtreg/compiler/arguments/TestTraceICs.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 8217447
  * @summary Test running TraceICs enabled.
```

Comment:
The summary is no longer applicable.

But really this test seems some what pointless. It basically checks that `Xlog:inlinecache` is a valid log setting, but doesn't actually check anything interesting.

Reply:
Updated. TestLogJIT is similar in only enabling the log setting without checking anything.

Reply:
Yes, these test could check output but it is separate issue. Let keep them and update later.